block/header.go

```diff
@@ -53,6 +53,22 @@ type headerBody struct {
 	Extension extension
 }
 
+func NewMockHeader() *Header {
```

Comment:
This function looks like something to be used only during testing. I would move it to `bandwidth_test.go` and make it private. To build the header I would take advantage of the block.Builder and extract the header. Something like
```
block := new(block.Builder).ParentID(thor.Bytes32{1}).Timestamp(1).GasLimit(100000).Beneficiary(thor.Address{1}).GasUsed(11234).TotalScore(1).StateRoot(thor.Bytes32{1}).ReceiptsRoot(thor.Bytes32{1}).Build()

header := block.Header()
```
obviously the builder logic can be responsibility of the moved function

Reply:
I see I didn't know you can use the builder.